atlp-rwanda/ecommerce-app-legends-bn
# #112 Issue admin view order

### What does this PR do?
This PR has the changes to fix the issue for the app functionalities that limit the users from listing the Orders to the admin and customer.

### How can this be manually tested ?
- clone this repo
- install the deps
- start the server `npm run dev`
- experience the app functionalities 
- browse the app on the server 5000 || 4000 and 
- login as customer via the swagger doc 
- add items to cart , checkout and pay then see the orders listed.
